packages/typespec-python/src/types.ts

```diff
@@ -300,6 +302,23 @@ function emitConstant(type: SdkConstantType) {
     });
 }
 
+function emitEnumValueType(context: SdkContext, type: SdkEnumValueType): Record<string, any> {
+    let valueType = undefined;
+    let emittedEnumName = undefined;
+    if (type.__raw && "enum" in type.__raw) {
```

Comment:
In the future, hope TCGC could add property `enum` for `SdkEnumValueType` : https://github.com/Azure/typespec-azure/issues/3711